examples/regression/1-quickstart/plot_regression_housing_dataset.py

```diff
@@ -0,0 +1,426 @@
+
+
+def sort(
+    y_test: NDArray, y_pred: NDArray, y_pis: NDArray
+) -> Tuple[NDArray, NDArray, NDArray, NDArray]:
+    """
+    Sorting the dataset such that you can make better plots.
+
+    Parameters
+    ----------
+    y_test : NDArray
+        _description_
+    y_pred : NDArray
+        _description_
+    y_pis : NDArray
+        _description_
+
+    Returns
+    -------
+    Tuple[NDArray, NDArray, NDArray, NDArray]
+        _description_
+    """
+    indices = np.argsort(y_test)
+    y_test_sorted = np.array(y_test)[indices]
+    y_pred_sorted = y_pred[indices]
+    y_lower_bound = y_pis[:, 0, 0][indices]
+    y_upper_bound = y_pis[:, 1, 0][indices]
+    return y_test_sorted, y_pred_sorted, y_lower_bound, y_upper_bound
+
+
+def plot(
+    title: str,
+    axs: plt.Axes,
+    y_test_sorted: NDArray,
+    y_pred_sorted: NDArray,
+    lower_bound: NDArray,
+    upper_bound: NDArray,
+    coverage: float,
+    width: float,
+    num_plots_idx: NDArray,
+) -> None:
+    """
+    Plotting of the different error bars, in red the test points
+    that fall outside of the error bars.
+
+    Parameters
+    ----------
+    title : str
+        _description_
+    axs : plt.Axes
+        _description_
+    y_test_sorted : NDArray
+        _description_
+    y_pred_sorted : NDArray
+        _description_
+    lower_bound : NDArray
+        _description_
+    upper_bound : NDArray
+        _description_
+    coverage : float
+        _description_
+    width : float
+        _description_
+    num_plots_idx : NDArray
+        _description_
+    """
+    axs.yaxis.set_major_formatter(FormatStrFormatter('%.0f'+"k"))
+    axs.xaxis.set_major_formatter(FormatStrFormatter('%.0f'+"k"))
+
+    lower_bound_ = np.take(lower_bound, num_plots_idx)
+    y_pred_sorted_ = np.take(y_pred_sorted, num_plots_idx)
+    y_test_sorted_ = np.take(y_test_sorted, num_plots_idx)
+
+    error = y_pred_sorted_-lower_bound_
+
+    warning1 = y_test_sorted_ > y_pred_sorted_+error
+    warning2 = y_test_sorted_ < y_pred_sorted_-error
+    warnings = warning1 + warning2
+    axs.errorbar(
+        y_test_sorted_[~warnings],
+        y_pred_sorted_[~warnings],
+        yerr=error[~warnings],
+        capsize=5, marker="o", elinewidth=2, linewidth=0
+        )
+    axs.errorbar(
+        y_test_sorted_[warnings],
+        y_pred_sorted_[warnings],
+        yerr=error[warnings],
+        capsize=5, marker="o", elinewidth=2, linewidth=0, color="red"
+        )
+    axs.scatter(
+        y_test_sorted_[warnings],
+        y_test_sorted_[warnings],
+        marker="*", color="green"
+    )
+    axs.set_xlabel("True house prices in $")
+    axs.set_ylabel("Prediction of house prices in $")
+    ab = AnnotationBbox(
+        TextArea(f"Coverage: {coverage}\nInterval width: {width}"),
+        xy=(np.min(y_test_sorted_)*3, np.max(y_pred_sorted_+error)*0.95),
+        )
+    axs.add_artist(ab)
+    axs.set_title(title, fontweight='bold')
+
+
+STRATEGIES = {
```

Comment:
You don't use `dict()` here ? :-)

Reply:
I think I will keep it as it is since this is how we setup the dictionary in other examples :)